build.gradle

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 import org.gradle.api.internal.plugins.osgi.OsgiHelper
```

Comment:
do we need this `org.gradle.api.internal.plugins.osgi.OsgiHelper`

Reply:
I removed it, but that made me realize that the test I was adding had to account for the case where a custom version is built with an additional version component 👍